vms/platformvm/block/parse_test.go

```diff
@@ -109,6 +109,7 @@ func TestProposalBlocks(t *testing.T) {
 			parentID,
 			height,
 			tx,
+			[]*txs.Tx{},
```

Comment:
We should test this field being set like the others

Reply:
I tend to agree with this, but really depends how D. wants to break up work.